elixir/apps/web/test/support/acceptance_case/vault.ex

```diff
@@ -98,7 +98,7 @@ defmodule Web.AcceptanceCase.Vault do
 
   def userpass_flow(session, oidc_login, oidc_password) do
     session
-    |> assert_text("Method")
+    |> find(Query.text("Sign in to Vault"))
```

Comment:
No issue with this change, but looking at the outputs of the CI pipeline for this branch, it looks like this may not have fixed the issue?

Reply:
Yeah, still trying to get it green. This became flaky when I added the icon to the Sign in buttons. Just a timing issue...

Reply:
Looks like I just needed to remove this altogether. Green now, hopefully for good